rllib/algorithms/bc/tests/test_bc_new_stack.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
```

Comment:
nit: same as above: Let's start flipping the namings from here on.

`test_..._new_stack` -> rename into `test_...`
`test_...` -> rename into `test_..._old_api_stack`

Additionally, in the BUILD file, we should put `#@OldAPIStack` above all old API stack tests. This way, little by little, we get a much better picture of what's still missing and what will eventually be deprecated.